crrExistingObjects.js

```diff
@@ -1,9 +1,17 @@
 const { doWhilst, eachSeries, eachLimit, waterfall, series } = require('async');
-const { Logger } = require('werelogs');
+const werelogs = require('werelogs');
 const { ObjectMD } = require('arsenal').models;
 const metadataUtil = require('./CrrExistingObjects/metadataUtils');
 
-const log = new Logger('s3utils::crrExistingObjects');
+const logLevel = Number.parseInt(process.env.DEBUG, 10) === 1
+    ? 'debug' : 'info';
+const loggerConfig = {
+    level: logLevel,
+    dump: 'error',
+};
+werelogs.configure(loggerConfig);
+const log = new werelogs.Logger('s3utils::crrExistingObjects');
+log.debug('debug');
```

Comment:
is this just a console.log at this point? Feels like it can be removed but not sure.

Reply:
Yes, it was a mistake. Removed.